README.md

```diff
@@ -32,29 +32,35 @@ Refer to [the documentation](https://hexdocs.pm/bridge_ex/BridgeEx.Graphql.html)
 
 If you need more control on your requests you can use [`BridgeEx.Graphql.Client.call`](https://hexdocs.pm/bridge_ex/BridgeEx.Graphql.Client.html#call/7) directly.
 
-#### Global configuration
+### Global configuration
 
 The following configuration parameters can be set globally for all bridges in the app, by setting them inside your `config.exs`:
 
 - `config :bridge_ex, auth0_enabled: true` to allow bridges to enable integration with Auth0
 - `config :bridge_ex, log_options: [log_query_on_error: true, log_response_on_error: false]` to customize logging in your bridges
 
-#### Authenticating calls via Auth0
+### Authenticating calls via Auth0
 
 `bridge_ex` supports authentication of machine-to-machine calls via Auth0, through the [prima_auth0_ex](https://github.com/primait/auth0_ex) library.
 
-To use this feature it is necessary to set the following configuration in your `config.exs`:
+To use this feature do the following:
 
-- auth0 support for the application in the environment must be enabled via `config :bridge_ex, auth0_enabled: true`.
-- configuration necessary to create API consumers with `prima_auth0_ex`, see [the documentation](https://github.com/primait/auth0_ex#api-consumer).
+- update your `config.exs`:
+
+  - enable auth0 support for the application in the environment: `config :bridge_ex, auth0_enabled: true`
+  - add the necessary config to create API consumers with `prima_auth0_ex`, see [the documentation](https://github.com/primait/auth0_ex#api-consumer)
+
+- add `:prima_auth0_ex` as a dependency in your mix project
 
 Then configure your bridge with the audience of the target service:
 
 ```elixir
-use BridgeEx.Graphql, endpoint: "...", auth0: [enabled: true, audience: "target_audience"]
+use BridgeEx.Graphql,
+  endpoint: "...",
+  auth0: [enabled: true, audience: "target_audience"]
```

Comment:
isn't this `enabled` redundant with the 'global' one?

Reply:
AFAIK it's implemented like this because the 'global' one is used to start `prima_auth0_ex` while the 'local' one then controls whether calls to Auth0 are made by the actual bridge to retrieve the `Authorization` header.

By removing the local flag we would make it so that `bridge_ex` couldn't be used on projects that call both Auth0 and API key authenticated endpoints, but then again we might want that

If `prima_auth0_ex` could be started "lazily" then we could retain only the local flag and then start `prima_auth0_ex` only if and when a bridge actually makes a call, but I don't know if it could be done and how

Reply:
I removed the global flag and made changes accordingly

Right now the process exits when a call is made and `prima_auth0_ex` is not started, which is kinda ugly but I don't know if and how it could be improved

Reply:
I guess we can do a check in the `BridgeEx.Graphql` `__using__` macro so that if you have `enabled: true` for `auth0` the compilation fails if you can't find `auth0_ex`

Reply:
(Dunno if it makes sense to you)

Reply:
I added a check with `Code.ensure_loaded?` which fails if `:prima_auth0_ex` is not present